pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -213,6 +219,8 @@ PipelineConfig createBatchOptions() {
         Instant.now().toString().replace(":", "-").replace("-", "_").replace(".", "_");
     options.setOutputParquetPath(dwhRootPrefix + TIMESTAMP_PREFIX + timestampSuffix);
 
+    options.setCreateParquetDwh(createParquetDwh);
```

Comment:
Have you tested the incremental pipeline when this flag is turned off. In particular does the `mergerPipelines` [here](https://github.com/google/fhir-data-pipes/blob/42c5e309b37faf6f61b0e5d3f993c6e7d448974c/pipelines/controller/src/main/java/com/google/fhir/analytics/PipelineManager.java#L681) work fine? I think we need extra logic in `PipelineManager` to handle these edge cases.